db/migrate/20230213234415_create_category_form_templates.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateCategoryFormTemplates < ActiveRecord::Migration[7.0]
+  def change
+    create_table :category_form_templates do |t|
+      t.references :category, foreign_key: true
+      t.references :form_template, foreign_key: true
```

Comment:
I think our internal consensus is to avoid using foreign keys (except when we know they'll be helpful). We can discuss this more internally as well, but I do want to raise it here because it's easier to decide _before_ merging the migration. cc @xfalcox

Reply:
Thanks for identifying this. This was added automatically when I ran the rails generator, but I've modified the migration to not include foreign keys. This table is likely similar in functionality to something like the `category_tags` table, so I'm following the pattern set there for now, but let's discuss more internally before merging the PR.

Reply:
Sounds good, thanks!